js/index.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function() {
+
+console.log('at doc ready');
+
+	$('.readless a').hide();
+	$('.readmore').click(articleOpen);
+	$('.readless').click(articleClose);
+	$('.learnmore').click(learnMoreOpen);
+
+
+	// article read more and read less
+	function articleOpen() {
+		$('#show-this-on-click').slideDown();
+		$('.readmore a').hide();
+		$('.readless a').show();
+	}
```

Comment:
^yup.  I changed the html from "#" to "#/" to prevent the reload of the page, but agree it'd be better to execute via js.